app/helpers/reports_helper.rb

```diff
@@ -44,4 +44,16 @@ def logs_show
              {}, {:class=>"col-md-1 form-control", :onchange =>"filter_by_level(this);"})
     end
   end
+
+  def report_origin_icon(origin)
+    return 'N/A' if origin.nil?
+    origin_icon = self.try("#{origin.downcase}_report_origin_icon".to_sym)
+    image_tag(origin_icon || origin + ".png", :title => _("Reported by %s") % origin)
+  end
+
+  def report_origin_output_partial(origin, default_partial = 'output')
```

Comment:
what is the purpose of `default_partial` being configurable? if I call this method and want to change the default, why do I call this method in the first place?

Reply:
It doesn't need to be passable, actually. It is called in the reports view and previously had only passed 'output'. 
I'll change it to simple return 'output' as a default for the render call in the view and remove the default_partial.